snuba/stateful_consumer/states/bootstrap.py

```diff
@@ -14,12 +22,60 @@ class BootstrapState(State[ConsumerStateCompletionEvent, ConsumerStateData]):
     control topic.
     """
 
+    def __init__(self,
+        topic: str,
+        bootstrap_servers: Sequence[str],
+        group_id: str,
+    ):
+        super(BootstrapState, self).__init__()
```

Comment:
```suggestion
        super().__init__()
```

Let's try to standardize on using the Python 3 nullary super moving forward, unless you can think of a reason why not to.